### PR TITLE
UPDATE: Support tags in <InputField />

### PR DESCRIPTION
**Pull request by tomashapl**
This Pull Request meets the following criteria:

* [ ] Tests have been added/adjusted for my new feature
* [ ] New Components are registered in index.js of my project
<br/><br/><br/><url>LiveURL: https://orbit-components-tag-in-input.surge.sh</url>

### CODE REVIEW

File: src/InputField/index.js [hunk 1]
```diff
@@ -198,14 +199,15 @@ export const Input = styled(
...
+  flex: 1 1 20%;
   width: 100%;
   height: 100%;
+  box-sizing: border-box;
   border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
   z-index: 2;
 
   // FIREFOX flexbox bug: the input doesn't shrink properly
-  min-width: 0;
+  min-width: 100px;
```
Comment (wEprous) [inline src/InputField/index.js]: Bug #756 appears when `100px`. Is it really needed when you setting `flex-basis: 20%`?

File: src/InputField/index.js [hunk 2]
```diff
@@ -265,6 +267,38 @@ const FormLabel = ({
...
+StyledInputTags.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledInputTagsInner = styled.div`
+  overflow-x: scroll;
+  white-space: nowrap;
+  
+  &::-webkit-scrollbar { 
+    display: none;
```
Comment (wEprous) [inline src/InputField/index.js]: Can you also hide scrollbar for Firefox?

**Comment by wEprous** [comment]
Also, can you add a small example to README.md? The source code is hidden in storybook 🙁

**Comment by tomashapl** [comment]
> Also, can you add a small example to README.md? The source code is hidden in storybook 🙁

👍

**Pull request merged**